### DISCUSSION TITLE
v10: uSync Config - Content Enabled by default

### DISCUSSION THREAD

**Discussion by KevinJump** [KevinJump/uSync#354] [Ideas]
In Umbraco v9 - we merged uSync and uSync.ContentEdition into one package, Without really thinking about it this had the effect of turning content and media syncing on by default. 

This can have effects, notably if your site is massive then uSync will start saving the content and media to disk, and that can be a) a lot of extra files b) slow a site down. 

So wondering if for the next major version (should we have one for v10) we should turn off Content and Media syncing and by default have settings only syncing ? 

Assumptions / Questions / Considerations ? 
- Most people don't actually want to sync content and media, so maybe it should be off ?
- You don't need content and media to be on for uSync.Publisher/Exporter to work (they use their own handlersets and in  v9+ they are defaulted to 'just work')
- Adding additional packages might need work, because if we enable 'just' settings, adding vendr would require changes to appsettings.json ? (we could do something about this)
- If we turn content/media off, should we also turn the dictionary on/off (its a 50/50 discussion for most are Dictionary Items content or settings?)

**Comment by leekelleher** [comment]
For initial development, e.g. at the time of install - I would have Content exporting enabled. Then once it's up on the content-entry environment, it's still be enabled, so that I could take a quick copy of content/media without a full on database sync/back-up. _(Of course, if I'd convinced them to purchase uSync Complete this wouldn't be necessary.)_ Once it's up on the production environment, I'd (generally) disable it.

If the feature was disabled by default, I'd simply enable it - wouldn't bother me that it was off to start with. I generally make other tweaks to the uSync config anyway. 😃

**Comment by NikRimington** [comment]
Personally, I think that content syncing should be off by default, except for Dictionary items that should be on and one-way by default (I argue that they are technically content but their initial creation is develop managed as they set up where they are used etc).

I didn't realise content handlers were turned on by default and it started saving dummy data into my repo (maybe I should have paid more attention to my committed files, but that is a separate issue :-D )

**Reply by KevinJump** [reply]
> I didn't realise content handlers were turned on by default and it started saving dummy data into my repo (maybe I should have paid more attention to my committed files, but that is a separate issue :-D )

yeah that's what i am worried about at the moment, if you don't know its on, there can be a lot of data and saving /checking that just isn't needed or wanted.
